theano/tensor/elemwise.py

```diff
@@ -125,6 +125,7 @@ class DimShuffle(Op):
 
     _f16_ok = True
     check_input = False
+    __props__ = ("input_broadcastable", "new_order")
```

Comment:
@nouiz I don't think i can test it with GPU. There is an issue with GPU while running the tensor's tests. ( https://github.com/Theano/Theano/issues/4131#issuecomment-190013710). I've currently tested the tensor's test with CPU, and sandbox/cuda/tests/ with GPU. After making the changes suggested by you, two tests fail. The failure is after I remove the opt, ```dimshuffle_as_view```. I've updated the PR with the commit, please check travis for the error. I tried debugging the failure message, but unable to figure out how. 
@abergeron , @lamblin : Since @nouiz is on vacation, I would be really glad if you could have a look at this :)